default.nix

```diff
@@ -48,6 +48,16 @@ rec {
     }
     // foldl recursiveUpdate { } (map (project: project.nixos.modules) (attrValues projects));
 
+  extendedNixosModules =
+    with lib;
+    [
+      nixos-modules.ngipkgs
+      # TODO: needed for examples that use sops (like Pretalx)
+      sops-nix
```

Comment:
Can we perhaps make the examples import the modules they need individually? This would make them more self-contained.

Reply:
Yes absolutely. IIUC&IIRC historically the reason to export them was that examples were built as full configurations, and collecting them at the flake level was practical to thread them into CI. We don't need that any more, the examples are just files we want people to copy and edit. The tests are still needed for CI but we can collect them out of the evaluated fix point.